packages/wallet/src/types.ts

```diff
@@ -45,6 +45,7 @@ export interface SyncStatus extends Shutdown {
 
 export type FinalizeTxProps = Omit<TxContext, 'signingContext'> & {
   tx: Cardano.TxBodyWithHash;
+  bodyCbor?: HexBlob;
```

Comment:
This change seems to be in the wrong commit. Also, how breaking would it be to replace `tx` and `bodyCbor` with `Serialization.Transaction`?

Reply:
I can try. Lets see

Reply:
This are the required changes: https://github.com/input-output-hk/cardano-js-sdk/pull/1321/commits/70246371e9f8c93b6d7f44ad1e8903cb7f6d2615

If we agree on this changes I will sort out the commits when squashing :)

Reply:
Changes looks good, but I just realized it's not going to work :pensive: We also need to update the extension messaging serializer/reviver (toSerializableObj/fromSerializableObj). Ideally it would be extensible so that we can inject API-specific serialization behavior to it. How about we tag this commit in some branch and create a ticket in LW-7386?

Reply:
done, and moved this change to the right commit